### ISSUE TITLE
Can't compile encfs with gcc 4.8.5

### ISSUE THREAD

**Issue by Vascom** [vgough/encfs#219]
Then I'm compile encfs 1.9.1 for EPEL 7 it use gcc 4.8.5 and cmake add parameter ";-std=c++11" to CXXFLAGS (parameter contain semicolon). It is brake compilation process:
https://kojipkgs.fedoraproject.org//work/tasks/4706/15704706/build.log

I am temporary solve problem by removing string
list(APPEND CMAKE_CXX_FLAGS "-std=c++11")
from CMakeLists.txt and add "-std=gnu++11" to CXXFLAGS in spec file.

Please correct it.

**Comment by isage** [comment]
It's not because of compiler.
list **never** should be used for manipulating flags, because, quote "A list in cmake is a ; separated group of strings".
It "works" just because the list is empty. However, when building from spec for fedora it's not empty (rpm sets CXXFLAGS), so two values are concatenated using ';'.

TL;DR:
`SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")`
should be used instead of
`list(APPEND CMAKE_CXX_FLAGS "-std=c++11")`

**Comment by vgough** [comment]
Thanks @isage .  Pushed fix to master.
